src/aircraft_cmd.cpp

```diff
@@ -399,9 +400,11 @@ ClosestDepot Aircraft::FindClosestDepot()
 		if (station == INVALID_STATION) return ClosestDepot();
 
 		st = Station::Get(station);
+		assert(st->airport.depot_id != INVALID_DEPOT);
 	}
 
-	return ClosestDepot(st->xy, st->index);
+	assert(st != nullptr);
```

Comment:
`st` can't be `nullptr` here, because if the branch wasn't taken `st` isn't `nullptr` and in the branch you already dereference `st`, so it can't be `nullptr` after that either.

Reply:
Ok